README.md

```diff
@@ -8,20 +8,21 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 ## Variables
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| aws_region | AWS region to use for resources. | string | - | yes |
-| cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `1024` | no |
-| memory | The amount of memory (in MiB) used to allocate for the Shunting Yard container. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `4096` | no |
-| docker_image | Full path to Shunting Yard Docker image. | string | - | yes |
-| docker_version | Shunting Yard Docker image version. | string | - | yes |
-| allowed_s3_buckets | List of S3 Buckets to which Shunting Yard will have read-write access. eg. `["bucket-1", "bucket-2"]` | list | `<list>` | no |
-| instance_name | Shunting Yard instance name to identify resources in multi-instance deployments. | string | `` | no |
-| subnets | ECS container subnets. | list | - | yes |
+| allowed\_s3\_buckets | List of S3 Buckets to which Shunting Yard will have read-write access. eg. `["bucket-1", "bucket-2"]`. | list | `n/a` | yes |
+| aws\_region | AWS region to use for resources. | string | n/a | yes |
+| cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `"1024"` | no |
+| ct\_common\_config\_yaml | Common Circus Train Config for internal Circus Train instance. Refer to [Circus-Train README](https://github.com/HotelsDotCom/circus-train/blob/master/README.md) for more details. | string | n/a | yes |
```

Comment:
This implies I can find something about the ct_common_config_yaml in the CT readme which isn't true perhaps something like:
`Common Circus Train Config file shared and added to all circus train runs started by shunting yard. Can for instance be used to configure the graphite settings.`

anyone else?

Reply:
Yeah. makes sense. How about this 

"Common Circus Train configuration to be passed to internal Circus Train instance. It can be used, for example to configure Graphite for Circus Train. Refer to [Circus-Train README](https://github.com/HotelsDotCom/circus-train/blob/master/README.md) for an exhaustive list of options supported by Circus Train."

Reply:
I want to keep the link to CT README so that a person can take a look at what options can be supported through this feature.

Reply:
ok sounds good